content/videos/git/1-fundamentals/5-command-line/index.json

```diff
@@ -0,0 +1,40 @@
+{
+    "title": "Intro to the Command Line",
+    "description": "In this fifth part of the Git and GitHub for Poets tutorial series, I give you a brief introduction to the Command Line and show you some of the most often used commands.",
+    "languages": [],
+    "topics": ["For beginners", "Git","GitHub"],
+    "videoId": "oK8EvVeVltE",
+    "timestamps": [
+        { "time": "0:00", "title": "Introduction" },
+        { "time": "1:13", "title": "The terminal prompt" },
+        { "time": "2:30", "title": "Basic commands" }
```

Comment:
Could add more sections, the whole video is basic commands so it might make sense to break it down further, maybe something like `Navigation`, `Tab completion`, `Command arguments`, etc